### PR TITLE
[2.11.x] DDF-3352 updated configuring data management docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates to the configuring section of the documentation; specifically configurations involving data management. 

#### Who is reviewing it? 

@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Review content. Most changes are just moving content, but some new material added.

#### What are the relevant tickets?
[DDF-3352](https://codice.atlassian.net/browse/DDF-3352)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/1580990/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/resources/content/_configuring/configuring-data-policy-plugins.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The <<Metacard Validity Filter Plugin>> determines whether metacards with validation errors or warnings are filtered from query results.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application.
+. Select the *Configuration* tab.
+. Select the *Metacard Validity Filter Plugin*.
+.. Check *Filter Errors* to hide metacards with errors from users.
+.. Check *Filter Warnings* to hide metacards with warnings from users.
+
+See <<ddf.catalog.metacard.validation.MetacardValidityFilterPlugin,Metacard Validity Filter Plugin configurations>> for all possible configurations.
```
Comment (ahoffer) [inline distribution/docs/src/main/resources/content/_configuring/configuring-data-policy-plugins.adoc]: Consider adding a reference to the MetacardValidityMarkerPlugin. It is the thing that adds the validation errors and warnings to the metacards.

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/configuring-data-policy-plugins.adoc]: will add link

**Comment by ricklarsen** [comment]
@ahoffer This is in the plugin section: https://github.com/codice/ddf/blob/master/distribution/docs/src/main/resources/content/_plugins/metacard-validity-marker.adoc 

Will add some configuration steps to this PR

**Comment by ricklarsen** [comment]
build now

**Pull request merged**